src/QuickMarcEditor/QuickMarcEditorContainer.js

```diff
@@ -104,17 +114,38 @@ const QuickMarcEditorContainer = ({
     const path = action === QUICK_MARC_ACTIONS.CREATE && marcType === MARC_TYPES.HOLDINGS
       ? EXTERNAL_INSTANCE_APIS[MARC_TYPES.BIB]
       : EXTERNAL_INSTANCE_APIS[marcType];
+    const isUserInMemberTenant = checkIfUserInMemberTenant(stripes);
+    const centralTenantId = stripes.user.user.consortium?.centralTenantId;
+
+    const requestOptions = {};
+
+    if (isUserInMemberTenant && action === QUICK_MARC_ACTIONS.DERIVE) {
+      const instanceRecord = await mutator.quickMarcEditInstance.GET({ path: `${path}/${externalId}` });
```

Comment:
This is sending request to get the same instance twice. Maybe we can pass `shared` as a prop from `ui-inventory` like `externalRecordPath`?

Reply:
Also, I think we need to pass the header when sending a POST request to derive, but I'm not 100% sure

Reply:
> This is sending request to get the same instance twice. Maybe we can pass shared as a prop from ui-inventory like externalRecordPath?

The `externalRecordPath` is a constant value and `shared` is a mutable value, we can only pass the state of `shared` through the search parameter.
> Also, I think we need to pass the header when sending a POST request to derive, but I'm not 100% sure.

If we don't specify the `headers`, they will be passed automatically and `x-okapi-tenant` will be `stripes.okapi.tenant`. We need to create a local record, so `x-okapi-tenant` should be a member tenant.

Reply:
> The externalRecordPath is a constant value and shared is a mutable value, we can only pass the state of shared through the search parameter.

Let's pass it as a search param - I'd really like to avoid sending two requests to get an instance record